packages/webpack/src/plugin.js

```diff
@@ -343,6 +357,11 @@ class LavaMoatPlugin {
             const chunkGraph = compilation.chunkGraph
 
             Array.from(chunks).forEach((chunk) => {
+              // Collect chunk IDs and info while we're here
+              if (chunk.id !== null) {
```

Comment:
I sense a weird, hard-to-uncover, bug around `chunk.id` being "of-false" without being specifically `null` sometime in the future... perhaps I'm wrong